test/fit_tests/tests/fileserver/test_microkernel_static_file_server.py

```diff
@@ -0,0 +1,368 @@
+'''
+Copyright 2016, DELL|EMC, Inc.
+
+Author(s):
+Norton Luo
+
+'''
+
+import os
+import sys
+import subprocess
+import urllib2
+import string
+import time
+import json
+import requests
+import random
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/fit_tests/common")
+import fit_common
+
+
+try:
+    MICROKERNEL_CONFIG = json.loads(open(fit_common.CONFIG_PATH + "fileserver_config.json").read())
+except:
+    print "**** Global Config file: " + fit_common.CONFIG_PATH + "fileserver_config.json" + " missing or corrupted! Exiting...."
+    sys.exit(255)
+test_microkernel_url=MICROKERNEL_CONFIG["microkernel"]
+
+# Local methods
+NODECATALOG = fit_common.node_select()
+
+
+
+def filerestful(url_command, rest_action='get', rest_payload=[], rest_timeout=None, sslverify=False, rest_headers={}):
+    '''
+    This function is a copy of the restful function used in fit_common. Due to the external file server used different authtication.
+    This routine executes a rest API call to the host.

```

Comment:
Initially I used fit_common to do this job but I then found there is two trouble:1. fit_common use ora IP to execute while the file server. And I need to access both servers in the bootstrapping test. 2. Static server do not have authentication but RackHD ora has authentication. I get trouble when call them together